.github/actions/push/action.yml

```diff
@@ -67,7 +69,7 @@ runs:
             repo: context.repo.repo,
             sha: '${{ inputs.head_sha }}',
             state: 'success',
-            context: context.workflow,
+            context: `${context.workflow} (${{ inputs.arch }})`,
```

Comment:
does the single `${` token trigger same substitution? just for consideration, a bit more expressive formatting:
```
${{ format('{0} ({1})', context.workflow, inputs.arch) }}
```